doc/source/image_description.rst

```diff
@@ -101,7 +101,7 @@ Image Includes
 
 .. code:: xml
 
-   <include from="filename.xml"/>
+   <include from="file://filename.xml"/>
```

Comment:
Crap, does this mean we probably want to re-introduce the `this://` URI handler as asked in #1964? Or are we going to automatically prepend `file://` if not specified?

Reply:
I think it would be a good idea to have this:// supported because it makes things explicit. But this should be done in another PR. So far you can still use the same settings as you had before, Examples:

1. from="file:///some/path/to/file.xml"
2. from="file://../../data/some/relative/path/to/file.xml"
3. from="/some/oath/to/file.xml"
4. from="../some/relative/path/to/file.xml"
 
So these are all accepted but the behavior is not really deterministic for 2. and 4.

What relative means is not clear. It depends on the pwd if no stylesheets worked on it, it depends on if you nest includes, it depends on if a schema version needs to be converted into a newer one because the converted result will come from /var/tmp

So overall I think with includes it should be clear, either by an absolute path name or by this:// which resolves into the image description directory

Thoughts ?

Reply:
So if I do `foo.xml`, it should resolve to `./foo.xml`, which resolves to `file://./foo.xml`, which resolves to the file relative to the working directory currently?

Reply:
yes, that's what also the unit test does.

Reply:
I realized that relative locations can be weird when the unit test which tested nested includes failed. So with this PR there are no behavior changes to what you saw before, we just have to realize that relative include locations can be weird under certain conditions and these conditions are not obvious to users imho